### ISSUE TITLE
Forêt : impossible de poser une arène des assassins

### ISSUE THREAD

**Issue by Jesst89** [JulienNigon/SpellcasterUniversity-Bug-Report#895]

**Comment by Neelan22** [comment]
Est ce que le souci de placement ne serait pas lié aux conditions de la salle? L'accès se fait par un couloir au fond de la salle ainsi que sur les cotés. Cela veut dire qu'il faut obligatoirement une salle à l'étage inférieur qui a elle même un passage par le fond et/ou qu'il y ait une salle à coté qui à une porte pour un passage horizontal.

**Comment by MlleMau** [comment]
Bonjour Jesst89 et Neelan22,

Personnellement, je n'ai jamais eu ce souci - cependant, je n'ai pas encore essayé la dernière MàJ ! Il est même possible de placer la salle au rez-de-chaussée, sans salle à côté. Il faut cependant 2 emplacements horizontaux. 
J'essaye cet après-midi Neelan, je vous dis quoi !

**Comment by Jesst89** [comment]
Hello Neelan22 et MlleMau,

Merci de vos réponses. Si ma mémoire est bonne, j'avais essayé de la poser au dernier étage : ce qui se passait à ce moment là c'est que l'emplacement apparaissait comme étant valide, mais en cliquant dessus il ne se passait rien, et je restais "bloquée" comme ça. Je ne pouvais plus cliquer ailleurs et j'ai été obligée de quitter le jeu avec ctrl-alt-suppr ... J'aurais dû être plus précise dans mon commentaire.

En tout cas bravo pour ce chouette jeu, j'adore le concept, les musiques, les différents univers, l'humour, l'ux bien pensée ... j'y ai passé un super moment (et j'y retourne aussitôt ^^) !

**Comment by MlleMau** [comment]
J'ai déjà rencontré un problème similaire un jour ! Comme je mettais un logiciel à jour en même temps, j'ai eu un lag, et la carte que j'essayais de placer s'est bloquée en cours de route. Plus moyen de la "rattraper" en appuyant dessus, et mon jeu restait bien évidemment en pause. J'avais une autre carte heureusement, que j'ai pu placer. Cela a débloqué mon jeu - mais ma carte bugguée est restée en minuscule sur mon écran, avec les paillettes qui défilaient quand je bougeais. Relancer la partie a suffit à récupérer cette carte.
Peut-être as-tu rencontré ce même problème ?

**Comment by Jesst89** [comment]
Non, je n'ai pas eu le problème de la carte qui restait buggée comme ça.
En revanche, j'ai pu poser une arène des assassins dans la forêt sans problème le lendemain, je pense que ça a été un simple lag aussi :)

**Comment by Neelan22** [comment]
oui on a des bugs liés à la sélection des cartes. On avait essayé de corrigé ça il y a 2 MaJ, mais il reste des petits soucis, tels que cité ici ou des double clics involontaires etc. Il faut qu'on revienne un peu dessus :/
